python/python/bystro/proteomics/tests/test_annotation_df.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
```

Comment:
Could you describe how this is testing annotation_df or change the name of it to improve clarity?

Reply:
yep, that's pretty unclear now that I look at it, refactored away